Benny 23 leadership views urls

bennytrieu opened the pull request:
- Tweaked the leadership model
1. title has been changed to a dropdown that uses the 10 different choices available in the TITLE_CHOICES array (I don't know if there are more ACM roles that I missed, I added these based on the [acmmanoa about page](https://acmmanoa.org/About/) and which person was which role. I also don't know the specific ordering except that President goes first and then Vice President goes next
3. made image field, mandatory to fill out
4. made bio field, optional to fill out
- Added leadership onto views and urls (Ordered by leadership title)
- Added Leadership onto navbar
- Added leadership.html

bennytrieu committed:
Added leadership onto views and URL, made a new leadership.html in pages, and passed leadership model data onto leadership.html. Also changed leadership model slightly

bennytrieu committed:
Made images smaller

bennytrieu committed:
Changed font styling of 'leadership' and 'faculty sponsers' a bit

jalenlum reviewed (changes requested):
Looks great overall! Just a few small changes to address before merging:

- Remove the `leadership.html` page and its corresponding URL.
- Test the model and view in `about.html` since that’s where it will likely be displayed. You can refer to the current ACM website for reference: [https://acmmanoa.org/About/](https://acmmanoa.org/About/)
- Change the plural name of the model to "Leadership." Currently it is showing as "Leaderships" in the admin portal. Remember to update migrations when you modify the model.

Let me know once these are done—everything else looks solid. Thanks!

bennytrieu committed:
Added requested changes

bennytrieu commented:
- Removed leadership.html and its url
- Moved the model and view to about.html
- Changed the name of the model from "Leaderships" to "Leadership in the admin portal and updated migrations.
(Funny thing is, when I was making the design of how the leadership and faculty staff will look on leadership.html, this link https://acmmanoa.org/About/   is the exact one I used as a reference lol.
